Corona-Warn-App/src/main/res/drawable/trace_location_view_cardhighlight_gradient.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:angle="135"
+        android:centerColor="#6C648C"
```

Comment:
small remark: some gradient xml files contain color references to the res->colors, others don't.
We should pay attention not to mix things up too much.
For me its ok this way..